### PR TITLE
feat(web): convo list

### PR DESCRIPTION
**Pull request by McPizza0**
Updates the convo list components and nav with new design

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#482** <a href="https://app.graphite.dev/github/pr/un/inbox/482?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by BlankParticle** [comment]
image
there are z-index issues with the avatar component

File: apps/web/src/components/avatar-plus.tsx [hunk 1]
```diff
@@ -1,103 +1,61 @@
...
-              className="absolute bottom-[-6px] right-[-4px] font-bold">
-              {`+${rest.length}`}
-            </Badge>
-          </button>
-        </Popover.Trigger>
-        <Popover.Content>
-          <div className="flex gap-2 p-2">
+    <div className="relative h-fit w-fit scale-100 overflow-visible">
+      <HoverCard>
+        <HoverCardTrigger>
```
Comment (BlankParticle) [inline apps/web/src/components/avatar-plus.tsx:28]: `HoverCardTrigger` is a anchor tag, using it inside other anchor tags will cause hydration errors. Add `asChild` attribute to it prevent that issue. we already have one hyrdation error in this PR

File: apps/web/src/app/[orgShortCode]/convo/layout.tsx [hunk 2]
```diff
@@ -1,24 +1,112 @@
...
+                  <CaretRight />
+                </BreadcrumbSeparator>
+                <BreadcrumbItem>
+                  <div className="bg-base-5 text-base-9 flex h-6 w-6 items-center justify-center rounded-sm">
+                    <ChatCircle
+                      weight="bold"
+                      className="h-4 w-4"
+                    />
+                  </div>
+                  <BreadcrumbLink href="/components">
```
Comment (BlankParticle) [inline apps/web/src/app/[orgShortCode]/convo/layout.tsx:83]: it href be `/${orgShortCode}/convos`

**Review by BlankParticle — APPROVED** [review]

**Comment by McPizza0** [comment]
### Merge activity

* **May 28, 8:06 AM EDT**: @McPizza0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/482).

**Pull request merged**